pallets/attestation/src/tests.rs

```diff
@@ -19,111 +19,98 @@
 use frame_support::{assert_noop, assert_ok};
 use sp_core::Pair;
 
-use crate::{self as attestation, mock::*};
+use crate::{self as attestation, mock::*, AccountIdOf, Config};
 use ctype::mock as ctype_mock;
 use delegation::mock as delegation_mock;
 
+// #############################################################################
 // submit_attestation_creation_operation
 
 #[test]
```

Comment:
S: A bit hard to read, but I haven't seen any tests that checks for the deposit after creation and after deletion?

Reply:
1e99025